### ISSUE TITLE
Update @all inside a form lead to javascript error.

### ISSUE THREAD

**Issue by ren-zhijun-oracle** [eclipse-ee4j/mojarra#4354]
[Stackoverflow](https://stackoverflow.com/questions/49071292/primefaces-and-jsf-2-3-javascript-error-cannot-read-property-error-of-null)

Seems like the component resources are available (see PrimeFaces HeadRenderer) in the postback but somehow they are not rendered in the response inside the head tag.

NOTE: this will only happen with update=@all as otherwise the head-tag won't be replaced. 

Discoverd with PrimeFaces 6.2 on Mojarra 3.3.2.

```
<h:form>
        <p:commandButton value="BUTTON_TEXT" update="@all"
            process="@none" />
    </h:form>
```

After pressing this button, this stacktraces shows the problem.

```
VM47:1 Uncaught TypeError: Cannot read property 'cw' of null
    at <anonymous>:1:12
    at p (jquery.js.xhtml?ln=primefaces&v=6.2:2)
    at Ja (jquery.js.xhtml?ln=primefaces&v=6.2:3)
    at r.fn.init.append (jquery.js.xhtml?ln=primefaces&v=6.2:3)
    at r.fn.init.<anonymous> (jquery.js.xhtml?ln=primefaces&v=6.2:3)
    at T (jquery.js.xhtml?ln=primefaces&v=6.2:3)
    at r.fn.init.html (jquery.js.xhtml?ln=primefaces&v=6.2:3)
    at Object.updateBody (core.js.xhtml?ln=primefaces&v=6.2:3)
    at Object.updateElement (core.js.xhtml?ln=primefaces&v=6.2:3)
    at Object.doUpdate (core.js.xhtml?ln=primefaces&v=6.2:3)
```

**Comment by tandraschko** [comment]
Issue with more description: https://github.com/javaserverfaces/mojarra/issues/4354

**Comment by tandraschko** [comment]
Duplicate of: https://github.com/eclipse-ee4j/mojarra/issues/4553

**Comment by rudi1602** [comment]
> * **Issue Imported From:** https://github.com/javaserverfaces/mojarra/issues/4354
> * **Original Issue Raised By:**@bigbonsai
> * **Original Issue  Assigned To:** Unassigned

Issue 4353 should not have been deleted, because it revealed connection between Issue and FullAjaxExceptionHandler and explained how jsf.js and primefaces.js differ and even providded js workaround.

**Comment by mnriem** [comment]
For Mojarra 2.3 and earlier please contact your vendor for support (RedHat, IBM, Oracle, Omnifish, Payara, etceteras)

**Comment by claudioweiler** [comment]
Hi!

This issue is closed as completed, but lacking info. I followed some links from #4553 (closed as stale) but I can't say for sure this is solved.

Is this solved for some 2.3.X version? Is this solved for 3.X or 4.X versions?

**Comment by melloware** [comment]
@claudioweiler please create a reproducer using 4.0.3 and if its an issue with 4.0.3 open a new ticket please with your reproducer.
